agent/gather.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# shellcheck source=/dev/null
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+source "$SCRIPTDIR"/agent/04_agent_configure.sh
```

Comment:
Sourcing a step script is usually not recommended and to be avoided, since it could run unwated pieces of code.  In this case it doesn't even seem to be necessary, a `source $SCRIPTDIR/common.sh` should be sufficient to access the OCP_DIR